modules/op-release-notes-1-13.adoc

```diff
@@ -140,3 +140,21 @@ include::snippets/technology-preview.adoc[]
 * Before this update, when using {tekton-chains}, if you configured the `storage.oci.repository` parameter, errors were reported in the Tekton Chains controller log. With this update, the `storage.oci.repository` parameter is processed correctly.
 
 * Before this update, when {tekton-chains} was configured with the Hashicorp Vault KMS and there was an issue with the connection to Vault, the {tekton-chains} controller pod could crash. With this update, the errors are processed and logged on the {tekton-chains} controller log.
+
+[id="release-notes-1-13-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.13.1
+
+With this update, {pipelines-title} General Availability (GA) 1.13.1 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-13-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run sometimes failed with a `cannot stop the sidecar` error message. With this update, the race condition between controllers that caused this failure is fixed.
+
+* Before this update, to enable keeping pods when a task run is cancelled, along with setting the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` in the `TektonConfig` CR, you also need to set the `pipeline.enable-api-fields` spec to `alpha` in the `TektonConfig` CR. With this update, setting the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` in the `TektonConfig` CR enables keeping pods when a task run is cancelled, and no additional setting is necessary.
+
+* Before this update, if you defined a sidecar for a task, {pipelines-shortname} did not validate the container image in the definition when creating the `Task` and `TaskRun` custom resources (CRs). At run time, a sidecar with an invalid container image caused the task run to fail. With this update, {pipelines-shortname} validates the container image in the sidecar definition when creating the `Task` and `TaskRun` CRs.
```

Comment:
Shouldn't it be "At run time, a sidecar with an invalid container image *would cause* the task run to fail" (or *is causing*) ?

Reply:
Reviewed, discussed with another senior writer. I think "would" would work if the "sidecar with an invalid container image" was a hypothetical prevented by validation. But because validation did not in fact happen, the sidecar was not fully hypothetical and actually did happen sometimes, which is why the current formula is probably better in this case.